src/samplers/cpu/stats.rs

```diff
@@ -41,3 +41,15 @@ gauge!(
 
 counter!(CPU_CYCLES, "cpu/cycles");
 counter!(CPU_INSTRUCTIONS, "cpu/instructions");
+gauge!(IPKC, "cpu/ipkc", "Average Instructions Per Thousand Cycle");
```

Comment:
@brayniac  Does gauge support the float numbers? Otherwise I have to use IPKC is to get better precisions.

Reply:
Gauge is only i64. So it would be better to use some scaled variant like you're doing with IPKC.